From 8ddd6a90ea3cbd3724191984039c71de8b58d41b Mon Sep 17 00:00:00 2001
From: Thomas Monjalon <thomas@monjalon.net>
Date: Fri, 27 Apr 2018 02:54:00 +0200
Subject: [PATCH 1/2] eal: fix build with glibc < 2.16

The fake getauxval function does not use its parameter.
So the compiler raised this error:
	lib/librte_eal/common/eal_common_cpuflags.c:25:25: error:
	unused parameter 'type'

Fixes: 2ed9bf330709 ("eal: abstract away the auxiliary vector")

Signed-off-by: Thomas Monjalon <thomas@monjalon.net>
Acked-by: Maxime Coquelin <maxime.coquelin@redhat.com>
---
 lib/librte_eal/common/eal_common_cpuflags.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/librte_eal/common/eal_common_cpuflags.c b/lib/librte_eal/common/eal_common_cpuflags.c
index a09667563..6a9dbaeb1 100644
--- a/lib/librte_eal/common/eal_common_cpuflags.c
+++ b/lib/librte_eal/common/eal_common_cpuflags.c
@@ -22,7 +22,7 @@
 
 #ifndef HAS_AUXV
 static unsigned long
-getauxval(unsigned long type)
+getauxval(unsigned long type __rte_unused)
 {
 	errno = ENOTSUP;
 	return 0;
-- 
2.17.0

